project.clj

```diff
@@ -19,5 +19,20 @@
             :url "http://opensource.org/licenses/Apache-2.0"}
   :dependencies [[org.clojure/clojure "1.7.0"]
                  [prismatic/schema "0.4.3"]]
-  :profiles {:test {:dependencies [[cheshire "5.5.0"]]
-                    :resource-paths ["resources" "test-resources"]}})
+  :aliases {"cljs-test" ["with-profile" "test-cljs" "test"]}
+
+  :profiles {:test      {:dependencies   [[cheshire "5.5.0"]]
+                         :resource-paths ["resources" "test-resources"]}
+             :test-cljs {:plugins      [[lein-cljsbuild "1.1.0"]]
+                         :dependencies [[org.clojure/clojurescript "1.7.48"]
+                                        [cheshire "5.5.0"]]
+                         :hooks        [leiningen.cljsbuild]
+                         :cljsbuild    {:builds        {:test
```

Comment:
More a code style nit. The alignment of the key/val isn't something that is following in the rest of the project. Would be good to keep it consistent